import-tags.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+public=''
+
+while getopts c:t:pu: flag
+do
+    case "${flag}" in
+        c) collection="${OPTARG}" ;;
+        t) file="${OPTARG}" ;;
+        p) public="-p" ;;
+        u) user="${OPTARG}"
+    esac
+
+uuid=$(./liquid dockerexec hoover:search ./manage.py getuuid $user | tail -1)
+./liquid dockerexec hoover:snoop -c $collection -t $file --uuid $uuid --user $user $public
```

Comment:
how do you feed the $file inside the hoover:snoop container? we have no shared mounts and we don't want to add any.

I think you need to change all the scripts to accept the CSV as std in.